.gitignore

```diff
@@ -55,3 +56,12 @@ yarn-error.log
 
 # CircleCI
 .circleci/src/workflows/@*.yml
+
+### VisualStudioCode ###
+.vscode/*
+*.code-workspace
+
+### VisualStudioCode Patch ###
+# Ignore all local history of files
+.history
+.ionide
```

Comment:
All these are not related to the project and should be in the .gitignore of your own profile in your laptop.